pkg/cloudprovider/aws/instance.go

```diff
@@ -103,7 +104,14 @@ func (p *InstanceProvider) launchInstance(ctx context.Context,
 	launchTemplateName string,
 	instanceTypeOptions []cloudprovider.InstanceType,
 	subnets []*ec2.Subnet,
-	capacityType string) (*string, error) {
+	capacityTypes []string) (*string, error) {
+
+	// Select Spot if within constarints
```

Comment:
Could you explain why we're doing this, in the comment? Right now the comment is just kinda saying the same thing.

Reply:
+1